backend/src/packages/auth/auth.controller.ts

```diff
@@ -6,8 +6,8 @@ import {
 } from '~/libs/packages/controller/controller.js';
 import { HttpCode } from '~/libs/packages/http/http.js';
 import { type ILogger } from '~/libs/packages/logger/logger.js';
-import { type UserSignUpRequestDto } from '~/packages/users/users.js';
-import { userSignUpValidationSchema } from '~/packages/users/users.js';
+import { type UserAuthRequestDto, type UserSignUpRequestDto } from '~/packages/users/users.js';
+import { userSignInValidationSchema, userSignUpValidationSchema } from '~/packages/users/users.js';
```

Comment:
These imports may be joined
```suggestion
import {
  type UserAuthRequestDto,
  type UserSignUpRequestDto,
  userSignInValidationSchema,
  userSignUpValidationSchema
  } from '~/packages/users/users.js';
```

Reply:
Also, to simplify, you can try following syntax - import type { UserAuthRequestDto, UserSignUpRequestDto,  userSignInValidationSchema, userSignUpValidationSchema } from '~/packages/users/users.js';
As far, as I remember, modern TS supports that, @andanf-e right?

Reply:
I think it (`import type { UserAuthRequestDto, UserSignUpRequestDto, userSignInValidationSchema, userSignUpValidationSchema } from '~/packages/users/users.js';`) won't work because like this you're declaring all import parts as types -- so this way schemas would be marked for TS also as types and it would cause errors

the main goal with such kind of import
```import {
  type UserAuthRequestDto,
  type UserSignUpRequestDto,
  userSignInValidationSchema,
  userSignUpValidationSchema
  } ```
with `type ` beginings for type imports is to tell TS earlier about things that may be skipped when it transpiles code to js